client/src/components/buildProfilePage/index.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import Ajax from '../../Ajax.js';
+import { connect } from 'react-redux';
+import { Alert, Form, Button } from 'react-bootstrap';
+import './style.scss';
+
+const mapStateToProps = (state) => {
+  return {
+    currentUser: state.user.currentUser
+  }
+}
+const mapDispatchToProps = (dispatch) => {
+  return {}
+}
+
+export default class BuildProfile extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      updated: false,
+      location: '',
+      // Adjust later - temporary hack to store skills and experience descripiton in one field
+      skill1: '',
+      experience: '',
+      skill: '',
```

Comment:
Unless I'm missing it, it looks like we're not using the skill property at all. We can probably get rid of skill1 and use the form to update skill directly. Then just concatenate the description onto skill in for the Ajax payload like you're doing.